### PR TITLE
give guidance to reduce scope of constants

### PR DESCRIPTION
**Pull request by tyler-french**
See https://github.com/uber-go/guide/issues/216 for more information.

This is just a draft, and more to start discussion to understand what the correct guidance here is.

### CODE REVIEW

File: src/var-scope.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Reduce Scope of Variables
+# Reduce Scope of Variables and Constants
```
Comment (tyler-french) [inline src/var-scope.md:1]: This might mess up existing links?

If so, we can just update description and examples

Comment (sywhang) [reply src/var-scope.md:1]: Yeah let's not edit the heading if possible. I think the description is pretty clear.

File: src/var-scope.md [hunk 2]
```diff
@@ -66,3 +66,36 @@ return nil
 
 </td></tr>
 </tbody></table>
+
+Avoid making constants global unless they are used in multiple functions.
```
Comment (sywhang) [inline src/var-scope.md:70]:
```suggestion
Constants do not need to be global unless they are used in multiple functions or files.
```

File: src/var-scope.md [hunk 3]
```diff
@@ -66,3 +66,36 @@ return nil
...
+
+</td><td>
+
+```go
+func Bar() {
+  const (
+    defaultPort = 8080
+    defaultUser = "user"
+  )
+  fmt.Println("Default port", _defaultPort)
```
Comment (abhinav) [inline src/var-scope.md:96]:
```suggestion
  fmt.Println("Default port", defaultPort)
```

File: src/var-scope.md [hunk 4]
```diff
@@ -66,3 +66,36 @@ return nil
 
 </td></tr>
 </tbody></table>
+
+Constants do not need to be global unless they are used in multiple functions or files.
```
Comment (abhinav) [inline src/var-scope.md:70]: optional qualification:

```suggestion
Constants do not need to be global unless they are used in multiple functions or files
or are part of an external contract of the package.
```

**Review by abhinav — APPROVED** [review]

**Pull request merged**